### ISSUE TITLE
openvpn should start **after** sshd

### ISSUE THREAD

**Issue by bodsch** [bodsch/ansible-openvpn#3]

**Comment by bodsch** [comment]
configure a sshd with to listeners:

`xxx.xxx.xxx.xxx` is the external IP
`yyy.yyy.yyy.yyy` is the VPN IP

```
ListenAddress xxx.xxx.xxx.xxx:AAAA
ListenAddress yyy.yyy.yyy.yyy:BBBB
```

With systemd, both services start in parallel, so that the bind to the VPN IP no longer works.

**Comment by bodsch** [comment]
this should fixed with commit 6131c43fa5a9ee3255f399f884598968cce29a8a
